packages/cli/test/unit/License.test.ts

```diff
@@ -252,4 +252,20 @@ describe('License', () => {
 			});
 		});
 	});
+
+	describe('reinit', () => {
+		it('should reinitialize license manager', async () => {
+			const license = new License(mock(), mock(), mock(), mock(), mock());
+			await license.init();
```

Comment:
This is already done in `beforeAll`?

Reply:
Not in this specific `describe` block 
![Screenshot 2024-05-21 at 09 29 22](https://github.com/n8n-io/n8n/assets/219272/706f3936-4809-4970-9136-1c4d4a60aadd)